packages/server/test/security/api.test.ts

```diff
@@ -49,32 +50,58 @@ const shouldFail = async (func: Function, onError: (err: AxiosError) => void) =>
 }
 
 describe('API', () => {
-  const sync = async (data?: SyncRequest, config?: AxiosRequestConfig) => {
-    const client = http()
+  describe('Admin', () => {
+    const createClient = async () => {
+      const client = http()
+      const res = await client.post('/api/admin/clients')
 
-    const res = await client.post<SyncResult>('/api/sync', data, config)
+      expect(res.data).toEqual({ id: expect.anything(), token: expect.anything() })
```

Comment:
We could consider, in the future, implementing some kind of expect.uuid util. What do you think?

Reply:
Could be useful. We could do that when revisiting the tests in another PR